Bpi/Sdk/Bpi.php

```diff
@@ -1,9 +1,21 @@
 <?php
-require_once __DIR__ . '/../../vendor/autoload.php';
+
+if (file_exists(__DIR__ . '/../../vendor/autoload.php')) {
+    require_once __DIR__ . '/../../vendor/autoload.php';
+}
```

Comment:
Is this required? I would prefer to keep `require_once` out of our classes [in accordance with PSR-1](http://www.php-fig.org/psr/psr-1/#side-effects).